CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+### v2.0.0-beta1 -> HEAD
```

Comment:
I believe we already talked about this offline, just repeating here: this should be `v2.0.0-beta1 -> v2.0.0-beta2`